docs/content/developers/release-management.md

```diff
@@ -110,16 +110,12 @@ While it’s more error-prone, images can be pushed from the command line:
 5. `make push VERSION=<release_version> DOCKER_ARGS=--no-cache` for most of the images. For `ddev-dbserver` it’s `make PUSH=true VERSION=<release_version> DOCKER_ARGS=--no-cache`. There’s a [push-all.sh](https://github.com/ddev/ddev/blob/master/containers/push-all.sh) script to update all of them, but it takes forever.
 6. `ddev-dbserver` images can be pushed with `make PUSH=true VERSION=<release_version> DOCKER_ARGS=--no-cache` from the `containers/ddev-dbserver` directory.
 
-## Maintaining `ddev-dbserver` MySQL 5.7 and 8.0 ARM64 Images
+## Maintaining `ddev-dbserver` MySQL 5.7 ARM64 Images
 
-Sadly, there are no ARM64 Docker images for MySQL 5.7 and 8.0, so we have our own process to maintain [ddev/mysql-arm64-images](https://github.com/ddev/mysql-arm64-images) and [ddev/xtrabackup-build](https://github.com/ddev/xtrabackup-build) images for DDEV.
+There are no ARM64 Docker images for MySQL 5.7, so we have our own process to maintain [ddev/mysql-arm64-images](https://github.com/ddev/mysql-arm64-images).
```

Comment:
To be clear, ARM64 for MySQL 5.7 that I see here is not what we want?
https://hub.docker.com/r/bitnami/mysql/tags?name=5.7

Reply:
I think it can be better stated. Seems the problem is we don't have a way to use xtrabackup with mysql 5.7 on arm64.